source/common/config/grpc_subscription_impl.cc

```diff
@@ -67,6 +67,12 @@ void GrpcSubscriptionImpl::onConfigUpdate(
             resources.size(), version_info);
 }
 
+void GrpcSubscriptionImpl::onConfigUpdate(
+    const Protobuf::RepeatedPtrField<envoy::service::discovery::v3::Resource>&,
+    const Protobuf::RepeatedPtrField<std::string>&, const std::string&) {
+  RELEASE_ASSERT(false, "delta onConfigUpdate called on GrpcSubscriptionImpl");
```

Comment:
nit: can you replace this with the NOT_REACHED macro (search code) and a comment? Thank you.

/wait